src/features/classification/Classification.js

```diff
@@ -36,20 +43,29 @@ const Classification = ({
     isLoadingControls,
     maxAppCount,
     messageStyle,
+    modifiedAt,
+    modifiedBy,
     name,
     itemName = '',
     color,
     onClick,
 }: Props) => {
     const isClassified = !!name;
     const hasDefinition = !!definition;
+    const hasModifiedAt = !!modifiedAt;
+    const hasModifiedBy = !!modifiedBy;
     const hasSecurityControls = !!controls;
     const isTooltipMessageEnabled = isClassified && hasDefinition && messageStyle === STYLE_TOOLTIP;
     const isInlineMessageEnabled = isClassified && hasDefinition && messageStyle === STYLE_INLINE;
     const isNotClassifiedMessageVisible = !isClassified && messageStyle === STYLE_INLINE;
     const isControlsIndicatorEnabled = isClassified && isLoadingControls && messageStyle === STYLE_INLINE;
     const isSecurityControlsEnabled =
         isClassified && !isLoadingControls && hasSecurityControls && messageStyle === STYLE_INLINE;
+    const isModifiedMessageVisible = isClassified && hasModifiedAt && hasModifiedBy && messageStyle === STYLE_INLINE;
+
+    const formattedModifiedAt = isModifiedMessageVisible && (
+        <FormattedDate value={new Date(modifiedAt)} {...formattedDateOptions} />
+    );
```

Comment:
Do we need to validate the date in this case? Maybe can use `isValidDate` from datetime utils here?